guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: guide
+title: Units & Symbols
+nav_order: 1
+parent: Payments
```

Comment:
@yantiarifin there is a new chapter called "Foundations" which Units & Symbols would now live in instead of Payments.

```suggestion
parent: Foundations
```